rfc6962-bis.xml

```diff
@@ -656,7 +655,7 @@ updated on the fly.
 (see <xref target="cert_sctlist_extension"/>). This mechanism allows the use of
 unmodified TLS servers, but, because the included SCTs cannot be changed without
 re-issuing the certificate, increases the risk that the certificate will be
-refused if any of the SCTs become invalid.
+considered non-compliant if some of the SCTs become invalid (by the client not recognizing the logs they were issued from anymore).
```

Comment:
Done - incorporated Rob's suggested text, dropping "with this document".
@benlaurie , any suggestion on where (in the document) to define compliance, if at all?